ngmt/modules/ssd/_pham.py

```diff
@@ -132,7 +136,17 @@ def detect(
 
         # Select gyro data and convert it to numpy array format
         gyro = data.iloc[:, 3:6].copy()
-        gyro = gyro.to_numpy()
+        gyro = -gyro.to_numpy()
```

Comment:
Why do you multiply with -1? Do we always need to do this, or does this depend on the sensor orientation?

Reply:
I handled this issue in the updated code and now there is no need to multiply by -1.

Reply:
How did you handle this? Do you have some examples?

Reply:
This isssue was handled here: [https://github.com/neurogeriatricskiel/KielMAT/blob/turning-detection-dev/ngmt/modules/ptd/_pham.py [265]](https://github.com/neurogeriatricskiel/KielMAT/blob/turning-detection-dev/ngmt/modules/ptd/_pham.py#L265)